codec/reflectcodec/type_codec.go

```diff
@@ -332,6 +352,60 @@ func (c *genericCodec) marshal(value reflect.Value, p *wrappers.Packer, maxSlice
 				return err
 			}
 		}
+		return nil
+	case reflect.Map:
```

Comment:
We should document that we can now serialize maps in the comment on `genericCodec` (specifically point 3 where we list types that are serializable)